File: SumNTerm.java
import java.util.Scanner;

public class APSum {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter first term: ");
        int a = sc.nextInt();

        System.out.print("Enter common difference: ");
        int d = sc.nextInt();

        System.out.print("Enter number of terms: ");
        int n = sc.nextInt();

        double sum = (n * (2 * a + (n - 1) * d)) / 2.0;

        System.out.println("Sum of first " + n + " terms = " + sum);

        sc.close();
    }
}

File: SumNTerm.cpp
#include <iostream>
using namespace std;

int main() {
    int a, d, n;
    float sum;

    cout << "Enter first term: ";
    cin >> a;

    cout << "Enter common difference: ";
    cin >> d;

    cout << "Enter number of terms: ";
    cin >> n;

    sum = (n * (2 * a + (n - 1) * d)) / 2.0;

    cout << "Sum of first " << n << " terms = " << sum;

    return 0;
}
